README.md

```diff
@@ -201,7 +224,7 @@ $ curl -Lo containerbuddy-0.0.4.tar.gz \
 $ tar -xf containerbuddy-0.0.4.tar.gz
 $ cp ./containerbuddy ./nginx/opt/containerbuddy/
 $ cp ./containerbuddy ./app/opt/containerbuddy/
-./start.sh -p example -f docker-compose-local.yml
+./run.sh -p example -f docker-compose-local.yml
```

Comment:
I can fix up this in the readme, but I don't have a good way of creating the non-local compose for etcd since it has some external dependencies for which I dont have access. 

Aside - the readme make reference to a specific version of Containerbuddy for the tarball. Should this be updated when the version gets bumped?